client/__tests__/e2e/puppeteer.setup.ts

```diff
@@ -29,6 +29,11 @@ beforeEach(async () => {
         status: 200,
         contentType: "application/json",
         body: JSON.stringify(DATASET_METADATA_RESPONSE),
+        // (thuang): Add headers so FE@localhost:3000 can access API@localhost:5000
+        headers: {
+          "Access-Control-Allow-Origin": "http://localhost:3000",
```

Comment:
Added these headers 🙆‍♂️

Reply:
Thanks Timmy!  :ok_woman:

Reply:
How do you run your tests locally? I am not seeing these errors when running `make smoke-test`.